src/main/java/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package service.impl;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import service.WriterService;
+
+public class WriterServiceImpl implements WriterService {
+    @Override
+    public void writeReport(String report, String fileName) {
+        File file = new File(fileName);
+        if (file.exists()) {
+            file.delete();
+        }
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't create file", e);
+        }
+        try {
+            Files.write(file.toPath(), report.getBytes(), StandardOpenOption.APPEND);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file " + fileName, e);
+        }
```

Comment:
```suggestion
        try {
            Files.write(new File(fileName).toPath(), report.getBytes());
        } catch (IOException e) {
            throw new RuntimeException("Can't write data to file " + fileName, e);
        }
```
`Files.write()` will do everything for you